### PR TITLE
ROLLUP-248/replace-zksync-with-rollup

### PR DESCRIPTION
**Pull request by franciscotobar**
## What

- Update all the messages with Rollup instead of zkSync

## Refs

- [ROLLUP-248](https://rsklabs.atlassian.net/browse/ROLLUP-248)

### CODE REVIEW

File: etc/test_config/sdk/test-vectors.json [hunk 1]
```diff
@@ -45,11 +45,11 @@
...
+            "pubKey": "9fce9db2059996381a66dcfbadca9f31629c04d894c4cefe137b2d29c95a5793",
+            "signature": "e9b5e8de5edd93310b8a9e874436435109f2ca96d8a10b05759c05fc40ebca96551404d14b8bfdb02045e52fe4a4575011a09dfd8e46eefe1b18f47aa4526e01"
           },
-          "ethSignMessage": "0x5472616e7366657220313030303030303030303030302e302045544820746f3a203078313961613265643837313230373265393138363332323539373830653538373639386566353864660a4665653a20313030303030302e30204554480a4e6f6e63653a203132",
-          "ethSignature": "0x4684a8f03c5da84676ff4eae89984f20057ce288b3a072605cbf93ef4bcc8a021306b13a88c6d3adc68347f4b68b1cbdf967861005e934afa50ce2e0c5bced791b"
+          "ethSignMessage": "0x5472616e7366657220313030303030303030303030302e30205242544320746f3a203078313961613265643837313230373265393138363332323539373830653538373639386566353864660a4665653a20313030303030302e3020524254430a4e6f6e63653a203132",
+          "ethSignature": "0x31667c153fd6fc477c2c5ee518ca247054b197d690dd5b5e53a5bb5ce39ddde5529f44141f68c7124c99cf17e1464255b0a76877726d6a8ee18c01694eba9c131b"
```
Comment (jurajpiar) [inline etc/test_config/sdk/test-vectors.json:52]: why did all these hashes change? Do the orig messages contain the changed word rollup?

**Review by jurajpiar — COMMENTED** [review]
Did you mean to leave many references in? Is this task meant to be done in stages to prevent thousands of lines for review?

From my notes:
- did you mean to exclude all *zksync* file names?
- what about the etc/env/\*/\*.toml?
- docs/\*.md
- Dockerfiles
- Event::ZkSyncWithdraw and other event names
Screenshot 2023-07-26 at 11 28 45

File: etc/test_config/sdk/test-vectors.json (hunk 1 above)
Comment (franciscotobar) [reply etc/test_config/sdk/test-vectors.json:52]: yes, since we changed contracts and those contracts generated signatures, it was necessary to change it.

**Comment by franciscotobar** [comment]
This task is related with messages and I quote "As a user, I don’t want to see zkSync related messages in Rollup, so that the system can appear consistent in the UX." , we can discuss during the daily but I understood that was just messages that were sent by the system, not all the zksync words.

File: contracts/contracts/Governance.sol [hunk 2]
```diff
@@ -170,7 +170,7 @@ contract Governance is Config {
         return
             abi.encodePacked(
                 "\x19Ethereum Signed Message:\n141",
-                "\nCreator's account ID in zkSync: ",
+                "\nCreator's account ID in rollup: ",
```
Comment (antomor) [inline contracts/contracts/Governance.sol:173]: I'd always use `Rollup` in upper case

File: core/bin/zksync_api/src/api_server/rpc_server/rpc_trait.rs [hunk 3]
```diff
@@ -96,8 +96,8 @@ pub trait Rpc {
     fn get_eth_tx_for_withdrawal(&self, withdrawal_hash: TxHash)
         -> BoxFutureResult<Option<String>>;
 
-    #[rpc(name = "get_zksync_version", returns = "String")]
-    fn get_zksync_version(&self) -> Result<String>;
+    #[rpc(name = "get_rollup_version", returns = "String")]
+    fn get_rollup_version(&self) -> Result<String>;
```
Comment (antomor) [inline core/bin/zksync_api/src/api_server/rpc_server/rpc_trait.rs:100]: I think that this is a bit out of topic, but it's fine, thank you 👍🏼

**Review by antomor — CHANGES_REQUESTED** [review]
@franciscotobar thank you for the changes, may I ask you to be consistent and always use `Rollup` (with the capital letter)

File: contracts/contracts/Governance.sol (hunk 2 above)
Comment (franciscotobar) [reply contracts/contracts/Governance.sol:173]: updated

**Review by antomor — APPROVED** [review]
LGTM

**Pull request merged**